### PR TITLE
Completed html-forms exercise.

### PR DESCRIPTION
**Pull request by samlee0258**
https://user-images.githubusercontent.com/110208589/214396725-9f23a18c-ec65-411e-b537-329fdf65f57d.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="text" name="email">
```
Comment (robgmerrill) [inline html-forms/index.html:12]: `input` should have attribute `id` with a value that matches the `label` value for `for` attribute.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        </div>
+      </div>
+      <div>
+        <label for="reason">Why is coding cool?</label>
+        <div>
+          <select name="reason">
```
Comment (robgmerrill) [inline html-forms/index.html:18]: `select` should have attribute `id` with a value that matches the `label` value for `for` attribute.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <option value="">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
```
Comment (robgmerrill) [inline html-forms/index.html:20]: Please use html entity `&apos;` instead of `'`. Lines 21 and 22 as well.

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+        <input type="radio" name="language" value="html"> HTML
```
Comment (robgmerrill) [inline html-forms/index.html:28]: Please have an individual label for each input on lines 28, 29, 30.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        <input type="radio" name="language" value="css"> CSS
+        <input type="radio" name="language" value="js"> JavaScript
+      </div>
+      <div>
+        <p>Would you like to sign up for more information?</p>
+        <input type="checkbox" name="signup"> Sign me up!
```
Comment (robgmerrill) [inline html-forms/index.html:34]: `Sign me up!` should be a label for the `input` checkbox.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+        <div>
+          <textarea name="comments" cols="30" rows="10"></textarea>
```
Comment (robgmerrill) [inline html-forms/index.html:39]: Add `id` that corresponds to the label's `for` attribute.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Just a few things to fill in and you'll be all set!

File: html-forms/index.html (hunk 1 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2023-01-24 at 1 42 01 PM

File: html-forms/index.html (hunk 2 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2023-01-24 at 1 42 34 PM

File: html-forms/index.html (hunk 3 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2023-01-24 at 1 43 10 PM

File: html-forms/index.html (hunk 4 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2023-01-24 at 1 43 46 PM

File: html-forms/index.html (hunk 5 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2023-01-24 at 1 44 16 PM

File: html-forms/index.html (hunk 6 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2023-01-24 at 1 44 42 PM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**